raylib/raylaz.cpp

```diff
@@ -29,7 +29,7 @@ bool readLas(const std::string &file_name,
 
   if (ifs.fail())
   {
-    std::cerr << "readLas: failed to open stream" << std::endl;
+    std::cerr << "readLas: failed to open file " << file_name << ", error: " << std::strerror(errno) << std::endl;
```

Comment:
I am not certain this error will inform if file does not exist.

Reply:
It does. Just tested it.
"readLas: failed to open file doesnottexist.blh, error: No such file or directory"